apps/explorer/views.py

```diff
@@ -1,15 +1,32 @@
+import re
+
 from django.db.models import Q
 from django.contrib import messages
 from django.contrib.auth.mixins import LoginRequiredMixin
 from django.http import HttpResponse, HttpResponseRedirect
 from django.urls.base import reverse
 from django.utils import timezone
+from django.utils.translation import ugettext as _
 from django.views.generic import DetailView, FormView, ListView
+from django.views.generic.detail import SingleObjectMixin
 from django.views.generic.edit import FormMixin
 
 from apps.core.models import PixelSet
-from .forms import PixelSetFiltersForm, PixelSetExportForm
-from .utils import export_pixelsets
+from .forms import (PixelSetFiltersForm, PixelSetExportForm,
+                    PixelSetExportPixelsForm)
+from .utils import export_pixelsets, export_pixels
+
+
+def str_to_set(input):
+    """Returns a set of strings by splitting the given `input` string on space,
+    comma or new line. It eliminates duplicates and strips each string.
+    """
+    return set(
+        filter(
+            None,
+            [part.strip() for part in re.split('\s*,\s*|\s+|\n', input)]
+        )
+    )
```

Comment:
I think `filter(None, iterable)` is harder to read/understand than the following:

```python
set([part.strip() for part in re.split('\s*,\s*|\s+|\n', input) if part])
```

WDYT?

I have to confess that I wasn't aware of this `None` function trick.

Reply:
It does not do the same thing, I want filtering after `strip()`

Reply:
I added a comment

Reply:
You are right: I missed the point.